apps/application-system/form/infra/application-system-form.ts

```diff
@@ -14,6 +13,22 @@ export const serviceSetup = (services: {}): ServiceBuilder<'application-system-f
         prod: 'https://innskra.island.is',
       },
     })
+    .secrets({
```

Comment:
This creates 7 new lists to manage, in 3 environments. That's 21 lists that we'll have to maintain.

Is there no way to give users credentials/scopes for this using ids?

Reply:
We are basing this of existing admin setups.
I belive the most correct way of dong this is using the delegation system (umboðskerfið), but I don't think that's ready yet.
Unless I'm missing something

Reply:
I know this is based off prior config - I'm only pointing out that the current state of things is crazy :(

Reply:
Agree this is suboptimal but I don't see a better solution here. 
Do you want us to change our approach here to something else?

Reply:
How often do you think these lists will change?

I don't have a better solution right now and it bothers me that I have to approve stuff because we can't do stuff properly, just good enough. Oh well.

Reply:
Thank you.
Once these list have been set up to include the correct nationalIds, I expect these lists to change every 4 years

Reply:
Hopefully we'll have a better way to do this in 4 years then 😅

Reply:
🤞